Library/Formula/macvim.rb

```diff
@@ -1,12 +1,12 @@
-# Reference: https://github.com/b4winckler/macvim/wiki/building
+require 'formula'
+
 class Macvim < Formula
   homepage 'https://code.google.com/p/macvim/'
-  url 'https://github.com/b4winckler/macvim/archive/snapshot-73.tar.gz'
-  version '7.4-73'
-  sha1 'b87e37fecb305a99bc268becca39f8854e3ff9f0'
-  revision 1
+  url 'https://codeload.github.com/macvim-dev/macvim/tar.gz/snapshot-74'
+  version '7.4-74'
+  sha1 '3c4fd307aff9b74700d3eece2a7b89942cd95cfb'
```

Comment:
Indeed. And change url to `https://github.com/macvim-dev/macvim/archive/snapshot-74.tar.gz`.